packages/cozy-stack-link/src/DocumentCollection.js

```diff
@@ -102,6 +101,36 @@ export default class DocumentCollection {
     }
   }
 
+  async create({ _id, _type, ...document }) {
+    const resp = await this.link.fetch(
+      'POST',
+      uri`/data/${this.doctype}/`,
+      document
+    )
+    return {
+      data: [normalizeDoc(resp.data, this.doctype)]
+    }
+  }
+
+  async update(document) {
+    const resp = await this.link.fetch(
+      'PUT',
+      uri`/data/${this.doctype}/${document._id}`,
+      document
+    )
+    return {
+      data: [normalizeDoc(resp.data, this.doctype)]
+    }
+  }
+
+  async destroy({ _id, _rev, ...document }) {
+    await this.link.fetch(
+      'DELETE',
+      uri`/data/${this.doctype}/${_id}?rev=${_rev}`
+    )
+    return
+  }
```

Comment:
Is this a good idea to make `destroy` a synchronous function when we could return the `fetch` result and let the caller of `destroy` to to something after the `DELETE` request?

Reply:
You're right, it's better to return the document with its updated _rev